devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -112,7 +112,7 @@ metadata {
 
 
 		main(["status", "acceleration", "temperature"])
-		details(["status", "acceleration", "temperature", "3axis", "battery", "refresh"])
```

Comment:
@tpmanley Ah, yeah, I can do that. The valueTile doesn't have to be left or anything for platform events to be generated, correct?